### ISSUE TITLE
Focus visible (WCAG 2.4.7)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1241]
**Request:** The purpose of this success criterion is to help a person know which element has the keyboard focus.

**Issues:**

1. https://slidewiki.org/ 
- Form field "Search for decks or people" is not highlighted by a focus when focused via keyboard.
- The menu entries in the user menu (page header) are not highlighted by visible focus. **(This also affects other pages!)**
--> When focusing, the fields should/could be highlighted with a visable focus frame. In the case of color highlighting, a sufficient contrast of 3:1 to surrounding colored areas must be ensured.

2. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
- Within the table of contents the focus highlighting is implemented via color change that does not meet the necessary contrast determinations. The buttons should be highlighted with a sufficiently contrasted frame (3:1). 

![grafik](https://user-images.githubusercontent.com/48785136/74634862-11c6a800-5165-11ea-989a-786ef94f17b3.png)

3. https://slidewiki.org/search?keywords=accessibility&sort=score
- Within the advanced search, the fields "Owners" and "Markierungen" are not highlighted by a keyboard focus.

**Comment by VivienDZB** [comment]
### Search - https://slidewiki.org/search?keywords=accessibility&sort=score
The form element "Markierungen" within the "Advanced Options" does not have a sufficiently contrasted focus frame. The contrast between the frame and the surrounding colored areas should be at least 3: 1.
![grafik](https://user-images.githubusercontent.com/48785136/95979769-71707280-0e1c-11eb-900a-679c90d11040.png)

### Editor - https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/edit
The buttons in the "Add" and "Edit" menus do not have a sufficiently contrasted focus frame. The contrast between the frame and the surrounding colored areas should be at least 3: 1.
![grafik](https://user-images.githubusercontent.com/48785136/95979855-949b2200-0e1c-11eb-84c1-c2124bb8c357.png)

**Comment by aoelen** [comment]
The tags input has been changed (replaced with the improved tags input which is also used at other locations): 

![image](https://user-images.githubusercontent.com/1567948/97019973-45a66880-1551-11eb-9dc1-51b9ba2b1c61.png)


The focus color for the buttons has been changed as well:

![image](https://user-images.githubusercontent.com/1567948/97019934-37584c80-1551-11eb-929b-9fb92b1809a7.png)
